dockstore-client/src/main/java/io/dockstore/client/cli/Client.java

```diff
@@ -102,6 +102,7 @@
     public static final AtomicBoolean DEBUG = new AtomicBoolean(false);
     public static final AtomicBoolean INFO = new AtomicBoolean(false);
     public static final AtomicBoolean SCRIPT = new AtomicBoolean(false);
+    public static final String DEPRECATED_PORT_MESSAGE = "Dockstore webservice has deprecated port 8443 and may remove it without warning. Please change it to use the standard 443 port via the Dockstore config file (\"~/.dockstore/config\" by default).";
```

Comment:
It would be nice to have an example, because 

1. It would be nicer if they didn't specify the port at all, i.e., encourage them to use the implicit port with https.
2. When you switch from port 8443, you have to add `/api`.